homeassistant/components/sensor/modbus.py

```diff
@@ -71,13 +84,14 @@ class ModbusSensor(Entity):
     # pylint: disable=too-many-arguments
     """ Represents a Modbus Sensor """
 
-    def __init__(self, name, slave, register, bit=None, unit=None):
+    def __init__(self, name, slave, register, bit=None, unit=None, coil=False):
         self._name = name
         self.slave = int(slave) if slave else 1
         self.register = int(register)
         self.bit = int(bit) if bit else None
         self._value = None
         self._unit = unit
+	self._coil = coil
```

Comment:
This indent looks like it was done with tabs instead of spaces. Please only use spaces for indentation.